db/types/base.py

```diff
@@ -9,3 +75,11 @@
 
 def get_qualified_name(name):
     return ".".join([preparer.quote_schema(SCHEMA), name])
+
+
+def get_installed_types(engine):
+    return engine.dialect.ischema_names
+
+
+def get_full_mathesar_type_name(name):
+    return f'mathesar_types.{name}'
```

Comment:
This should be either replaced by the `get_qualified_name` function, or wrap it (or replace it wherever that function is used).  Otherwise, we risk these getting out of sync, leading to bugs.

Reply:
Thanks, fixed. I opted to use `get_qualified_name` everywhere because it's more robust, it looks at the schema.